manifests/w/WHONET/AMRInterpretationEngine/23.6.21/WHONET.AMRInterpretationEngine.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: WHONET.AMRInterpretationEngine
+PackageVersion: 23.6.21
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{87B936E6-717F-410D-80DE-8C8E129A52ED}'

```

Comment:
```suggestion
ProductCode: '{87B936E6-717F-410D-80DE-8C8E129A52ED}'
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
```

Reply:
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/42c8bd00-ec56-473f-818e-d3970d7124a2)

Reply:
Hi @drshtmstry,

This package appears to require this dependency. We'll be accepting new packages with dependencies soon, after the 1.6 release.